homeassistant/components/velux/config_flow.py

```diff
@@ -0,0 +1,71 @@
+"""Config flow for Velux integration."""
+from typing import Any
+
+from pyvlx import PyVLX, PyVLXException
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_HOST, CONF_PASSWORD
+from homeassistant.data_entry_flow import FlowResult
+import homeassistant.helpers.config_validation as cv
+
+from .const import _LOGGER, DOMAIN
+
+
+class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for NINA."""
+
+    VERSION: int = 1
+
+    @staticmethod
+    async def test_connection(host: str, password: str):
+        """Test the connection to Velux."""
+        pyvlx = PyVLX(host=host, password=password)
+        await pyvlx.connect()
+        await pyvlx.disconnect()
+
+    async def async_step_import(self, config: dict[str, Any]) -> FlowResult:
+        """Import a config entry."""
+        self._async_abort_entries_match({CONF_HOST: config[CONF_HOST]})
+        return self.async_create_entry(
+            title="VELUX",
+            data=config,
+        )
+
+    async def async_step_user(
+        self, user_input: dict[str, Any] | None = None
+    ) -> FlowResult:
+        """Handle the initial step."""
+        errors: dict[str, Any] = {}
+
+        if self._async_current_entries():
+            return self.async_abort(reason="single_instance_allowed")
```

Comment:
Why make this single instance?
Are users not allowed to have two gateways (on separate IP)?